tfhe/src/integer/server_key/radix_parallel/tests_unsigned/mod.rs

```diff
@@ -59,6 +45,54 @@ pub(crate) const MAX_VEC_LEN: usize = 25;
 #[cfg(tarpaulin)]
 pub(crate) const MAX_VEC_LEN: usize = 5;
 
+/// Returns th number of loop iteration within randomized tests
+///
+/// The bigger the number of bits bootstrapped by the input parameters, the smaller the
+/// number of iteration is
+pub(crate) const fn nb_tests_for_params(params: PBSParameters) -> usize {
+    let full_modulus = params.message_modulus().0 * params.carry_modulus().0;
+
+    if cfg!(tarpaulin) {
+        // Use lower numbers for coverage to ensure fast tests to counter balance slowdown due to
+        // code instrumentation
+        1
+    } else {
+        // >= 8 bits (4_4)
+        if full_modulus >= 1 << 8 {
+            return 5;
+        }
+
+        // >= 6 bits (3_3)
+        if full_modulus >= 1 << 6 {
+            return 15;
+        }
+
+        30
```

Comment:
Looks much better
How did you come up with these numbers?

Reply:
Arbitrary choice somewhat based on the timing of the PBS :grin: